docs/source/image_process.mdx

```diff
@@ -0,0 +1,154 @@
+# Process image data
+
+🤗 Datasets support loading and processing images with the [`Image`] feature. This guide will show you how to:
+
+- Load an image dataset.
+- Load a generic image dataset with `ImageFolder`.
+- Add data augmentations to your images with [`Dataset.set_transform`].
+- Use [`~Dataset.map`] to quickly apply transforms to an entire dataset.
+
+## Image datasets
+
+The images in an image dataset are typically either a:
+
+- PIL `image`.
+- Path to an image file you can load. 
+
+For example, load the [Food-101](https://huggingface.co/datasets/food101) dataset and take a look:
+
+```py
+>>> from datasets import load_dataset, Image
+
+>>> dataset = load_dataset("food101", split="train[100:]")
```

Comment:
Why `train[100:]` and not just `train` ? (sorry if you've already discussed this)

Reply:
I wanted to load and test with a smaller number of examples first, but we can use the full `train` if you'd like!

Reply:
Then it should be `train[:100]` no ? :P